memphis88/week4/ass5/app/mixins/on-pressing-enter-mixin.jsx

```diff
@@ -0,0 +1,7 @@
+var OnPressingEnterMixin = {
```

Comment:
I think it seems overkill to have a mixin for this, you could just write one line to check for the keycode of <Enter>.